### PR TITLE
Add Image Picker Prop

### PR DESCRIPTION
**Pull request by sebko**
Also updates React (for the latest version of material-ui) and Enzyme (to fix tests)

**TODO:**
- [ ] Update mira-elements version after merging https://github.com/mirainc/raydiant-elements/pull/307

### CODE REVIEW

File: examples/menu/package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
     "prop-types": "^15.6.1",
     "raydiant-kit": "^4.1.1",
     "raydiant-scripts": "^4.1.1",
-    "react": "^16.4.0",
-    "react-dom": "^16.4.0"
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0"
```
Comment (sebko) [inline examples/menu/package.json]: the latest version of material ui uses `React.memo` so I've upgraded all our packages to use React 16.8.0

File: packages/raydiant-kit/package.json [hunk 2]
```diff
@@ -26,8 +26,8 @@
     "@types/prop-types": "^15.5.4",
     "@types/react": "^16.4.8",
     "@types/react-dom": "^16.0.7",
-    "enzyme": "^3.3.0",
-    "enzyme-adapter-react-16": "^1.1.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
```
Comment (sebko) [inline packages/raydiant-kit/package.json]: Tests broke after upgrading React. Upgrading enzyme and enzyme-adaptor-react-16 fixed the issue.

File: packages/raydiant-kit/src/prop-types/index.ts [hunk 3]
```diff
@@ -22,6 +23,7 @@ export const facebookAuth = () => new FacebookAuthType();
 export const file = (label: string) => new FileType(label);
 export const googleAuth = () => new GoogleAuthType();
 export const image = (label: string) => new ImageType(label);
+export const imagePickerField = () => new ImagePickerFieldType();
```
Comment (jschr) [inline packages/raydiant-kit/src/prop-types/index.ts]: Can we call this `images`? or `selectionWithImages`?

Comment (jschr) [reply packages/raydiant-kit/src/prop-types/index.ts]: Or alternatively combine this with `selection` and have a `.imagesUrl` method on `SelectionType`.

**Pull request merged**